dockstore-common/pom.xml

```diff
@@ -46,6 +46,11 @@
             <version>${postgresql.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.24</version>
```

Comment:
Version probably redundant (should be managed in the root pom.xml since we use snakeyaml in a few places)